reports/filesDataExporter/handlers.go

```diff
@@ -0,0 +1,172 @@
+package filesDataExporter
+
+import (
+	"bytes"
+	"context"
+	"sort"
+
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/strfmt"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/gen/storage/client/operations"
```

Comment:
could not import github.com/iryonetwork/wwm/gen/storage/client/operations (cannot find package "github.com/iryonetwork/wwm/gen/storage/client/operations" in any of:
	/tmp/tmp.9iNHKsROgt/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/gen/storage/client/operations (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/gen/storage/client/operations (from $GOROOT)
	/tmp/tmp.9iNHKsROgt/src/github.com/iryonetwork/wwm/gen/storage/client/operations (from $GOPATH))